DecipherNow/helm-charts
# Data chart cleanup

Corresponding PR to #511 with data chart

Changes

- Use a different proxy kubernetes label name. Instead of app, we're using “gm.proxy”
- Update proxy port to use 10808
- Use snake case
- Add comments to all values files
- Expose all envvars in values files, remove all possible envvars from template files
- Clean up all global values

To test:

1. Install secrets

```bash
helm dep up secrets
helm install secrets secrets -f credentials.yaml
```

2. Install the data chart

```bash
helm dep up data
helm install data data --set=global.environment=kubernetes
```

Make sure all of the pods come up and are configured as desired

3. Check out the values files comments https://github.com/DecipherNow/helm-charts/blob/data-values/data/values.yaml
